apps/src/templates/CoursePromoCard.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { PropTypes, Component } from 'react';
+import Radium from 'radium';
+import color from "../util/color";
+import Button from './Button';
+
+const styles = {
+  card: {
+    overflow: 'hidden',
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: color.border_gray,
+    position: 'relative',
+    height: 440,
+    width: 310,
+    marginBottom: 20,
+    backgroundColor: color.white
+  },
+  image: {
+    width: 310,
+    height: 220,
+    backgroundColor: color.purple
+  },
+  text: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.charcoal,
+  },
+  title: {
+    padding: 20,
+    fontSize: 20,
+  },
+  button: {
+    margin: 20,
+  },
+  description: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    fontSize: 14,
+    lineHeight: 1.5,
+  },
+  shareLink: {
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: color.border_gray,
+    fontSize: 16,
+    marginLeft: 20,
+    marginRight: 20,
+    marginTop: 10,
+    padding: 5,
+    width: 258,
+  },
+  ltr: {
+    float: 'left',
+  },
+  rtl: {
+    float: 'right',
+    textAlign: 'right'
+  },
+};
+
+/**
+ * A card used on /congrats to display information about a particular course
+ * includes an image. Not to be confused with CourseCard, the larger component used on /home for scripts and courses specific to a user.
+ */
+class CoursePromoCard extends Component {
```

Comment:
I wonder if we want to name this `TwoRowActionBlock` (to match [`TwoColumnActionBlock`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx)) since it may be used for more than just courses.